src/react/6-stateful-hooks.md

```diff
@@ -1,28 +1,28 @@
 @page learn-react/stateful-hooks Stateful Hooks
 @parent learn-react 6
 
-@description Learn how to use hooks in React to manage the state of a component.
+@description Learn how to use React's hooks to manage the state of a component.
 
 @body
 
 ## What are stateful hooks?
 
-Stateful hooks are React's way of managing all of the data that a functional component needs to keep track of. The most widely used stateful hook, `useState`, streamlines the process of both using and updating individual state values in a straightforward way.
+Stateful hooks are React's way of managing the data that a functional component needs to remember. The most widely used stateful hook, [`useState`](https://reactjs.org/docs/hooks-state.html), streamlines the process of both using and updating individual state values in a straightforward way.
```

Comment:
I know this is a nit-pick, but I personally take issue with attributing the ability to "remember" to components. JavaScript itself "remembers" them by keeping them in memory. React just tracks to where the references lead and updates them as necessary.

Reply:
I definitely think we should link to the useState docs here. Slack me to talk through the nit-pick but I really don't see an issue with using "remember" here.